controllers/remote/cluster_cache_tracker.go

```diff
@@ -346,11 +345,11 @@ func (t *ClusterCacheTracker) deleteAccessor(ctx context.Context, cluster client
 		return
 	}
 
-	log.V(2).Info("Deleting clusterAccessor", "cluster", cluster.String())
-
-	log.V(4).Info("Stopping cache", "cluster", cluster.String())
+	t.log.WithValues("cluster", klog.KRef(cluster.Namespace, cluster.Name))
```

Comment:
should this be 
```suggestion
	log := t.log.WithValues("cluster", klog.KRef(cluster.Namespace, cluster.Name))
```
and then use log